solutions/swb-reference/integration-tests/tests/isolated/environments/start.test.ts

```diff
@@ -31,6 +34,27 @@ describe('environment start negative tests', () => {
   });
 
   describe('ITAdmin tests', () => {
+    test('environment in failed state', async () => {
+      const failedEnvId = settings.get('failedEnvId');
+      const defaultProjId = settings.get('projectId');
+      try {
+        await itAdminSession.resources.projects
+          .project(defaultProjId)
```

Comment:
I'd suggest you take a look at paabHelper. it helps create PaabResources (projects for PA1, PA2 etc.). Avoid creating new project in the test env. In this case you can test against pa1Session and use project1Id.

Reply:
Ah yes, I looked at `paabHelper` and I see that it create new projects. However, in this test, we can't use preexisting projects because we need an environment in the `failed` state. 

I don't think there's a straight forward programmatic way to create an environment that is in the `FAILED` state, so I went with the option that the dev should preset an environment in the failed state. I saw that we are already requiring devs to set up a default project in the config file. This code will now read in the default project value.

Let me know if this is ok.

Reply:
@SanketD92 , @maghirardelli , @kpark277 , @aws-tyler thoughts?

Reply:
I've move the tests to multistep env test, and used paabHelper